gold-digger/src/resolvePromise.js

```diff
@@ -17,7 +17,7 @@ function resolvePromise(promiseToResolve, promiseState, setState){
     }
 
     if (promiseState.promise === null) return;
-    promiseToResolve.then(saveDataACB).catch(saveErrorACB);
+    return promiseToResolve.then(saveDataACB).catch(saveErrorACB);
```

Comment:
Not necessary, since it doesn't return anything.

Reply:
Yup, will fix, I had an unfortunate thought where I thought I could include it in a promisechain